src/test/regress/expected/citus_local_table_triggers.out

```diff
@@ -0,0 +1,422 @@
+---------------------------------------------------------------------
+ another_trigger         | "interesting!schema"."citus_local!_table"         | O
+ another_trigger_1507006 | "interesting!schema"."citus_local!_table_1507006" | O
+ trigger\'name22         | "interesting!schema"."citus_local!_table"         | O
+ trigger\'name22_1507006 | "interesting!schema"."citus_local!_table_1507006" | O
+ truncate_trigger_xxxxxxx  | "interesting!schema"."citus_local!_table"         | O
+(5 rows)
+
+DROP TRIGGER another_trigger ON "interesting!schema"."citus_local!_table";
+NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1507006, 'interesting!schema', 'DROP TRIGGER another_trigger ON "interesting!schema"."citus_local!_table";')
+DROP TRIGGER "trigger\'name22" ON "interesting!schema"."citus_local!_table";
+NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1507006, 'interesting!schema', E'DROP TRIGGER "trigger\\''name22" ON "interesting!schema"."citus_local!_table";')
+-- show that drop trigger works as expected
+SELECT * FROM citus_local_table_triggers;
+         tgname         |                  tgrelid                  | tgenabled
+---------------------------------------------------------------------
+ truncate_trigger_xxxxxxx | "interesting!schema"."citus_local!_table" | O
+(1 row)
+
+BEGIN;
+    CREATE TRIGGER "another_trigger\'name"
+    AFTER TRUNCATE ON "interesting!schema"."citus_local!_table"
+    FOR EACH STATEMENT EXECUTE FUNCTION dummy_function();
+    ALTER TABLE "interesting!schema"."citus_local!_table" DISABLE TRIGGER "another_trigger\'name";
+    -- show that our truncate trigger is disabled ..
+    SELECT * FROM citus_local_table_triggers;
+         tgname         |                  tgrelid                  | tgenabled
+---------------------------------------------------------------------
+ another_trigger\'name  | "interesting!schema"."citus_local!_table" | D
+ truncate_trigger_xxxxxxx | "interesting!schema"."citus_local!_table" | O
+(2 rows)
+
+    ALTER TABLE "interesting!schema"."citus_local!_table" ENABLE TRIGGER ALL;
+NOTICE:  executing the command locally: SELECT worker_apply_shard_ddl_command (1507006, 'interesting!schema', 'ALTER TABLE "interesting!schema"."citus_local!_table" ENABLE TRIGGER ALL;')
+    -- .. and now it is enabled back
+    SELECT * FROM citus_local_table_triggers;
+         tgname         |                  tgrelid                  | tgenabled
+---------------------------------------------------------------------
+ another_trigger\'name  | "interesting!schema"."citus_local!_table" | O
+ truncate_trigger_xxxxxxx | "interesting!schema"."citus_local!_table" | O
+(2 rows)
+
+ROLLBACK;
+-- as we create ddl jobs for DROP TRIGGER before standard process utility,
+-- it's important to see that we properly handle non-existing triggers
+-- and relations
+DROP TRIGGER no_such_trigger ON "interesting!schema"."citus_local!_table";
```

Comment:
can you do rename trigger for a non-existing trigger?

Reply:
no, just wanted see we let standard process utility to safely error out for such cases (as we process Drop Trigger commands in pre process)

Reply:
Do we need to add such a test ? The reason I added that test for `drop trigger` is that we process them in pre process, but we process rename trigger commands after standard process utility executes